PROJECT

```diff
@@ -34,7 +34,7 @@ resources:
   domain: openshift.io
   group: ptp
   kind: PtpOperatorConfig
-  path: github.com/openshift/ptp-operator/api/v1
+  path: github.com/k8snetworkplumbingwg/ptp-operator/api/v1
```

Comment:
Should these be replaced?

Reply:
Would it be worth writing a script to replace these using a regex or something like that?

Reply:
We want to keep downstream using the same thing as upstream here, so I don't think any reason for that.